README.md

```diff
@@ -23,350 +22,80 @@
-- **Built-in HAR converter**: record browser sessions as [`.har` files](https://en.wikipedia.org/wiki/.har) and [directly convert them to k6 scripts](https://k6.io/docs/using-k6/session-recording-har-support)
-- **Flexible metrics storage and visualization**: [InfluxDB](https://k6.io/docs/results-visualization/influxdb-+-grafana) (+Grafana), [JSON](https://k6.io/docs/getting-started/results-output/json) or [k6 Cloud](https://k6.io/docs/cloud/analyzing-results/overview)
-- [**Cloud execution**](https://k6.io/docs/using-k6/cloud-execution) and distributed tests _(currently only on infrastructure managed by [us k6 cloud](https://k6.io/cloud), with native distributed execution in k6 [planned](https://github.com/grafana/k6/issues/140) for the near future!)_
-
-There's even more! [See all features available in k6.](https://k6.io/docs/)
-
-Install
--------
-
-### Mac
-
-Install with [Homebrew](https://brew.sh/) by running:
-
-```bash
-brew install k6
-```
-
-### Windows
-
-If you use the [Chocolatey package manager](https://chocolatey.org/) you can install the unofficial k6 package with:
-
-```
-choco install k6
-```
-
-Otherwise, you can manually download and install the [latest official `.msi` package](https://dl.k6.io/msi/k6-latest-amd64.msi).
-
-### Linux
-
-For Debian-based Linux distributions like Ubuntu, you can install k6 from the private deb repo like this:
-
-```bash
-sudo apt-key adv --keyserver hkps://keyserver.ubuntu.com:443 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
-echo "deb https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
-sudo apt-get update
-sudo apt-get install k6
-```
-
-If you have issues adding the key from the keyserver, you can instead run:
-```bash
-curl -s https://dl.k6.io/key.gpg | sudo apt-key add -
-```
-
-Then confirm that the key with the above ID is shown in the output of `sudo apt-key list`.
-
-And for rpm-based ones like Fedora and CentOS:
-
-```bash
-sudo dnf install https://dl.k6.io/rpm/repo.rpm    # use yum instead of dnf for older distros
-sudo dnf install k6    # use yum install --nogpgcheck k6 for older distros (e.g. CentOS 7, Amazon Linux 2) without PGP V4 signature support
-```
+- **Configurable load generation.** Even lower-end machines can simulate lots of traffic.
+- **Tests as code.** Reuse scripts, modularize logic, version control, and integrate tests with your CI.
+- **A full-featured API.** The scripting API is packed with features that help you simulate real application traffic.
+- **An embedded JavaScript engine.** The performance of Go, the scripting simplicity of JavaScript.
```

Comment:
Nit: I wouldn't say the benefits of JS are in its simplicity (Go is arguably a simpler language). The decision to script in JS was mostly because of its widespread use and familiarity.